docs/UserGuide.md

```diff
@@ -31,104 +31,158 @@
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find b*y`<br>
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+> * Returns `buy` but not `buy eggs`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+* `find b*y*`<br>
 
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+> * Returns `buy eggs` but not `must buy eggs`
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Deleting a task : `delete`
+Deletes the specified task.<br>
+Format: `delete INDEX/TASKNAME`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+> * Deletes the task at the specified index
+> * If TASKNAME is entered instead, all tasks with matching names are displayed along with their indices. The user can then proceed with deletion using the index of the appropriate task.
 
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+Examples:
+* `delete 1`
+
+> * Deletes task at index 1
+
+* `delete b*y*`
+
+> * Lists tasks matching `b*y*` for the user to choose from.
+
+#### Updating a task : `update`
+Updates information for a task referred by its index.<br>
+Format: `update INDEX [TASKNAME] [at/from [START_TIME][START_DATE]] [to/by [END_TIME][END_DATE]] [PRIORITY]`
+
+> * Replaces the entered information for the task at the specified index
+> * If updated START_TIME/DATE exceeds END_TIME/DATE of the task, the update will not be successful
+
+Examples:
+* `update 1 at 13/09/2016 5pm`<br>
+
+> * `Meeting from 13/09/2016 4pm to 13/09/2016 6pm` will be replaced with `Meeting at 13/09/2016 5pm`
+
+* `update 1 from 13/09/2016 4pm to 13/09/2016 6pm`
+
+> * `Meeting at 13/09/2016 5pm` will be replaced with `Meeting from 13/09/2016 4pm to 13/09/2016 6pm`
+
+#### Marking a task as complete : `done`
+Marks a task as complete.<br>
+Format: `done INDEX/TASKNAME`
+
+> * Marks the task at the specified index as complete
+> * If TASKNAME is entered, tasks are sought out in the same way the `find` command does. Matching names and their indices are then displayed. The user can then proceed with marking a task as complete using the index of the appropriate task.
+
+Examples:
+* `done 1`
+* `done b*y*`
+
+#### Setting the storage location: `setstorage`
```

Comment:
This feature is not so important, because most of users do not care about the saving location. 
Letting users ware of the storge location may intoduce extra instability to your app. For example, users may delete or modify the data by mistake.
Alternatively, exporting data to a specific location can be a more useful and safe feature.